addons/web_editor/static/src/js/editor/snippets.editor.js

```diff
@@ -2164,11 +2164,11 @@ var SnippetsMenu = Widget.extend({
         this.$('.o_snippet_search_filter').toggleClass('d-none', tab !== this.tabs.BLOCKS);
         this.$('#o_scroll').toggleClass('d-none', tab !== this.tabs.BLOCKS);
         this.customizePanel.classList.toggle('d-none', tab === this.tabs.BLOCKS);
-        this.textEditorPanelEl.classList.toggle('d-none', tab !== this.tabs.OPTIONS);
+        this.textEditorPanelEl.classList.toggle('d-none', tab !== this.tabs.THEME);
 
         this.$('.o_we_add_snippet_btn').toggleClass('active', tab === this.tabs.BLOCKS);
-        this.$('.o_we_customize_snippet_btn').toggleClass('active', tab === this.tabs.OPTIONS)
-                                             .prop('disabled', tab !== this.tabs.OPTIONS);
+        this.$('.o_we_customize_snippet_btn').toggleClass('active', tab === this.tabs.THEME)
+                                             .prop('disabled', tab !== this.tabs.THEME);
```

Comment:
MOU changed those line recently, could you rebase the branch and fix conflicts? :)